bin/install

```diff
@@ -172,7 +172,7 @@ get_download_url() {
   esac
   local -r releases_host="https://releases.hashicorp.com"
 
-  if ! curl -fs "${releases_host}/${toolname}/${version}/${filename}" >/dev/null && [[ ${filename} == *"arm64"* ]]; then
+  if ! curl -fsI "${releases_host}/${toolname}/${version}/${filename}" >/dev/null && [[ ${filename} == *"arm64"* ]]; then
```

Comment:
Can we expand this to long form options, e.g., `--head`? It's a lot more legible in script contexts.

```suggestion
  if ! curl --fail --silent --head "${releases_host}/${toolname}/${version}/${filename}" >/dev/null && [[ ${filename} == *"arm64"* ]]; then
```

Reply:
All happy with that!